device-types/Eaton/5PX3000RTN.yaml

```diff
@@ -0,0 +1,29 @@
+manufacturer: Eaton
+model: 5PX3000RTN
+slug: 5px3000rtn
+u_height: 2
+is_full_depth: true
+power-outlets:
+  - name: Group1 - Outlet1
+    type: nema-5-20r
+  - name: Group1 - Outlet2
+    type: nema-5-20r
+  - name: Group2 - Outlet1
+    type: nema-5-20r
+  - name: Group2 - Outlet2
+    type: nema-5-20r
+  - name: Primary Group - Outlet1
+    type: nema-5-20r
+  - name: Primary Group - Outlet2
+    type: nema-5-20r
+  - name: L5-30R
+    type: nema-5-30r
+power-port:
+  - name: L530R
+    type: nema-5-30p
+device_bays:
+  - name: EBM
```

Comment:
This device type does not appear to house child devices. (Remember that a child device is one with its own independent control/management plane.)

Reply:
Removed bay